### ISSUE TITLE
Kitematic w/ Docker Toolbox on Windows can't find Docker Machine

### ISSUE THREAD

**Issue by carlin-q-scott** [docker/kitematic#2050]
### Expected behavior
Kitematic should launch without issue
### Actual behavior
1. Kitematic tries to connect to docker via a pipe and fails
2. I click "Use Virtualbox"
3. Kitematic instantly displays this error message: 

> Docker Machine is not installed. Please install it via the Docker Toolbox.

### Information about the Issue
I installed Docker Toolkit which came with Kitematic.
Docker version 1.12.0, build 8eab29e
I am using a standard user account but I tried run Kitematic as admin with even less success.

**Comment by FrenchBen** [comment]
can you verify that docker-machine installed properly?
via a powershell run: `docker-machine version`

**Comment by carlin-q-scott** [comment]
Yes, I can use it from a command prompt. I even used it to manage some docker containers to make sure it was working. I did have to manually setup the environment variables it requires to run which is tracked in a different issue.

**Comment by FrenchBen** [comment]
Did you specify a different installation folder perhaps than usual?
In the code we set the following env var `DOCKER_TOOLBOX_INSTALL_PATH` which indicates where everything was installed. 
Can you confirm that this env var was set properly?

**Comment by carlin-q-scott** [comment]
That environment variable was not set. Adding it has resolved the issue. 

The problem with the installation of Toolbox is that it appears to use `set` rather than `setx` during installation, the former only sets an environment variable for the current shell session. I mentioned this in docker/toolbox#484 as well.

**Comment by FrenchBen** [comment]
Closing this in favor of the toolbox issue. Thanks for helping us troubleshoot this.

**Comment by popopome** [comment]
If any suggested solutions didn't work for you, try this.
Physically remove the settings by deleting a folder.

`C:\Users\<YOUR_USER_NAME>\AppData\Roaming\Kitematic
`

**Comment by serkanodaci** [comment]
Thanks @popopome :)

**Comment by Tamiltam0007** [comment]
Encountered an error: Error: Command failed: C:\Program Files\Docker Toolbox\docker-machine.exe status default,open C:\Users\ELCOT\.docker\machine\machines\default\config.json: The system cannot find the file specified.,  ......how to resolve this.....plz reply

**Comment by kiviev** [comment]
The same thing happened to me. At this moment I am using Windows Home.

At least in my case, what happened was that the environment variables The same thing happened to me. At this moment I am using Windows Home.

At least in my case, what happened was that the environment variables DOCKER_MACHINE and DOCKER_TOOLBOX_INSTALL_PATH were not created for the system.

I just had to add them and it worked. and **DOCKER_TOOLBOX_INSTALL_PATH** were not created for the system.

I just had to add them and it worked.

![docker machine env error](https://user-images.githubusercontent.com/5745777/40055373-fff01a34-5847-11e8-8f94-617ad537bcd9.JPG)
